frontend/test/metabase/visualizations/components/settings/ChartSettingFieldsPicker.unit.spec.js

```diff
@@ -0,0 +1,71 @@
+import React from "react";
+import userEvent from "@testing-library/user-event";
+import { renderWithProviders, screen } from "__support__/ui";
+
+import ChartSettingFieldsPicker from "metabase/visualizations/components/settings/ChartSettingFieldsPicker";
+
+const DEFAULT_PROPS = {
+  options: [
+    { name: "Count", value: "count" },
+    { name: "Average of Total", value: "avg" },
+  ],
+  colors: { avg: "#A989C5", count: "#509EE3" },
```

Comment:
are we really hard-coding the colors like this?
shouldn't we use whitelabelable colors?